### DISCUSSION TITLE
GPN Partnumbers

### DISCUSSION THREAD

**Discussion by tnemrap** [inventree/InvenTree#6716] [General]
Is someone using Generic Part Numbers?
https://www.eurocircuits.com/generic-components/generic-part-numbers/
If someone is i would be interested how you did it.
It would be nice to have a partnumber for 10k 1/5w 1% resistors and under that is different manufacturers and suppliers.

**Comment by matmair** [comment]
I am not aware of any usage of this but this would certainly be an interesting plugin
